src/featureFlags/flags/avalanche.ts

```diff
@@ -0,0 +1,15 @@
+import { ChainId } from '@uniswap/sdk-core'
+
+import { BaseVariant, FeatureFlag, useBaseFlag } from '../index'
+
+export function useAvalancheFlag(): BaseVariant {
+  return useBaseFlag(FeatureFlag.avalanche)
+}
+
+export function useIsAvalancheEnabled(): boolean {
+  return useAvalancheFlag() === BaseVariant.Enabled
+}
+
+export function filterChainsForAvalanche(chains: ChainId[]): ChainId[] {
+  return chains.filter((chain) => chain !== ChainId.AVALANCHE || useIsAvalancheEnabled())
+}
```

Comment:
I'm confused on how this function is able to use a hook... is useIsAvalancheEnabled not actually a react hook?

wouldn't this need to be something like
```suggestion
export function useFilterChainsForAvalanche(chains: ChainId[]): ChainId[] {
  const isAvalancheEnabled = useIsAvalancheEnabled()
  return chains.filter((chain) => chain !== ChainId.AVALANCHE || isAvalancheEnabled)
}
```

but clearly it's compiling so I must be missing something here?

Reply:
I've updated the fn to follow hook naming conventions